### DISCUSSION TITLE
Step down DC-DC component

### DISCUSSION THREAD

**Discussion by 4lphac** [philibertc/micronova_controller#72] [General]
Hi, I've just bought the kit,  also I have these in house, I suppose they'll work too right? 

https://it.aliexpress.com/item/4000038240358.html?spm=a2g0o.order_list.order_list_main.5.104836968tacFt&gatewayAdapt=glo2ita

Do I need to provide 3.3V or 5V (USB power) like stated in the the tindie mail? From the docs I see 3.3v is expected.

One more thing, I've got a pretty standard Anselmo Cola, is there anything I should know?

Thanks!

**Comment by philibertc** [comment]
Hello,
yes this DC-DC module should work, even if mini modules like these sometimes produce a not very clean voltage, causing crashes of the esp8266. So give it a try.
> I've got a pretty standard Anselmo Cola, is there anything I should know

No, I can't think of anything special to say about this stove.

Bests!
Philibert

**Reply by kubrick89** [reply]
Hi,
I think 5V pin on D1 is connected to the onboard voltage regulator, why can't we use 5V from the stove directly on 5V pin of the D1? Any particular reason I'm missing?

Thanks,
Marco

**Reply by poupounetjoyeux** [reply]
Hello @philibertc,

I ordered your PCB and starting to check how I will plug it on my stove
I'm self asking the same question as @kubrick89 :
Is there a reason why the 5V pin of the stove is not used to power the weemos using its 5V pin?
If not do you think it's a possible way to remove the need of a buck converter ?
 (I have the original micronova wifi module that I want to replace by your solution to bypass the cloud API and it seems it uses only 3 wires so for me it can be possible, i added a photo)
If you think it's possible, it means I need to jump the 5V from stove in pins to the 5V pin on the PCB ? 

![IMG_20250525_102935_868](https://github.com/user-attachments/assets/b8d52ec2-0707-490b-a6eb-794bad0d9607)

Thanks for your great work!
I'm so impatient to receive my PCB 😁

**Reply by poupounetjoyeux** [reply]
Just saw your post on another thread https://github.com/philibertc/micronova_controller/discussions/17#discussioncomment-1489946

So I think it's possible at least with my stove regarding my current wifi board but by doubt and to protect it, I will opt for the buck converter and try to make a 3D printed box for it (will share it when finished 😉)

**Reply by poupounetjoyeux** [reply]
Hello @philibertc 
Me again, with another issue,

After testing my pins I have found the ground, 12V (yes not 20V in my case), but the two others are 3.3V and 5V
I plugged a logic analyzer and it seams that 3.3V is the serial
Is this meaning that I need to plug the 3.3V from my buck converter instead of the 5V in the "from stove" 5V pin?
